README.md

```diff
@@ -1,33 +1,39 @@
+# ROS-bonsai-connector
+This project assumes that the user has a Robot Operating System (ROS) based simulation and are looking to use it to train a reinforcement learning policy. It is recommended that the user have a basic understanding of ROS or ROS2 before proceeding with the **Integrating Your Simulation** section.
+
+An a fully Bonsai-integrated Turtlebot3 simulation has been provided as an example. See the following section for a link to its README.
```

Comment:
"An a fully..." => "A fully..."

Reply:
"See the following section for a link to its README" => `See the [Turtlebot3 Sample](#turtlebot3-sample) section.`

Reply:
Implemented feedback. Thanks!